core/artifact-core-hipinfo.toml

```diff
@@ -0,0 +1,6 @@
+# hipInfo
+[components.run."core/hipInfo-ds/stage"]
+optional = "windows"
+include = [
+  "bin/hipInfo.exe",
```

Comment:
If this used a pattern (`bin/**`) instead of `hipInfo.exe`, we could also include this on Linux. Fine to start with just Windows though.